i3_agenda/__init__.py

```diff
@@ -38,6 +38,8 @@ def main():
     args = parser.parse_args()
 
     events = load_events(args)
+    if args.skip > 0:
+        events = events[args.skip :]
```

Comment:
This could cause an exception after too many clicks (we don't query all events), it will raise an index out of bound exception when the skip was clicked n times where the user only got n-1 events queried

Reply:
Just tried to pass a `--skip 1000000` and the `get_closest` function just returns "No events"; no problem to add a try/except if you prefer

Reply:
my bad then :)